src/modules/Services/Twilio-Whatsapp.ts

```diff
@@ -0,0 +1,89 @@
+import TagoIOModule, { GenericModuleParams } from "../../common/TagoIOModule";
+
+interface TwilioWhatsappData {
+  /**
+   * Number to send Whatsapp message, Example: +5599999999999
+   */
+  to: string;
+  /**
+   * Message to be send
+   */
+  message: string;
+  /**
+   * From number registered with Twilio, Example: +5599999999999
+   */
+  from: string;
+  /**
+   * Twilio account SID
+   */
+  twilio_sid: string;
+  /**
+   * Twilio auth token
+   */
+  twilio_token: string;
+  /**
+   * Content/Type of the request when sending a file, must be the same content type of the file
+   */
+  content_type?: string;
+  /**
+   * Public URL of the file to be sent
+   */
+  media_url?: string;
```

Comment:
It must match the attachment format from the email service.

Reply:
done